README.md

```diff
@@ -213,12 +213,12 @@ what to put for each step.
 1. The elasticsearch instance domain can be one that is prexisting if
    used only for development.
 
-1. Next you will need a Google Client ID and Client secret. If you
-   missed it before, instructions to set this up are found on the
-   [Boardwalk deployment page](boardwalk/README.md#deployment).
-   _Boardwalk_ has functionality to export metadata to Broad's
-   FireCloud. In order to use it you need to provide your Google Cloud
-   Platform credentials.
+1. Provide the Google Client ID and Client Secret you got from the
+   OAuth2 app. Instructions to set this up can be
+   found on the [Boardwalk deployment page](boardwalk/README.md#deployment).That will enable user to log in with a Google account
+   (username + pwd) to work with access-controlled data if needed.
```

Comment:
I would remove this whole line, and put a period at the end of the previous line.

1. `(username + pwd)`, which should have `password` spelled out if we kept it, is not necessarily complete, as you may also have 2 factor auth.
2. I think most people will know what logging in with Google means, so we don't need to describe it.
3. I don't think we need to get into the details of what logging in enables you to do here, especially since we let you log in even if there is not access controlled data.